.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
We can add these to the Gitpod file. Would that be good?

Reply:
It would be great.